ct/hoarder.sh

```diff
@@ -40,7 +40,10 @@ function update_script() {
     msg_ok "Stopped Services"
     msg_info "Updating ${APP} to v${RELEASE}"
     cd /opt
-    mv /opt/hoarder/.env /opt/.env
+    if [[ -f /opt/hoarder/.env ]] && [[ ! -f /etc/hoarder/hoarder.env ]]; then
+      mkdir -p /etc/hoarder
+      mv /opt/hoarder/.env /etv/hoarder/hoarder.env
```

Comment:
There's a typo here.

```suggestion
      mv /opt/hoarder/.env /etc/hoarder/hoarder.env
```